rebasehelper/plugins/build_log_hooks/files.py

```diff
@@ -256,75 +258,130 @@ def _correct_missing_files(cls, rebase_spec_file, files):
         return result
 
     @classmethod
-    def _correct_deleted_files(cls, rebase_spec_file, files):
+    def _get_line_directives(cls, split_line) -> Tuple[List[str], Optional[str]]:
+        """Gathers directives present in the line.
+
+        Args:
+            split_line: Line split in spaces. Each element must either be
```

Comment:
Shouldn't it be split at spaces? And I think whitespace(s) would be more accurate.